harness/determined/estimator/_callback.py

```diff
@@ -0,0 +1,29 @@
+import pathlib
+
+
+class EstimatorCallback:
+    """
+    Abstract base class used to define a callback that should execute during
+    the lifetime of a EstimatorCallback.
+
+    .. warning::
+        If distributed or parallel training is enabled, only the chief GPU (rank = 0)
+        will execute the `on_checkpoint_end` callbacks.
+    """
+
+    def on_checkpoint_load(self, checkpoint_dir: pathlib.Path) -> None:
+        """
+        Run when checkpoint is loaded at startup. If not resuming from checkpoint
+        this is never called.
+        """
```

Comment:
"at startup" -> "when the task environment container starts up"

Reply:
Done